### PR TITLE
2020 HTTP/2 edits

### PR DESCRIPTION
**Pull request by MikeBishop**
Mostly spelling and lots of commas. Progress on #921

### CODE REVIEW

**Review by rviscomi — COMMENTED** [review]
These changes all look good at first glance. Thanks for correcting them. Could you also confirm if you're happy with `MikeBishop_bio`?

**Comment by MikeBishop** [comment]
Taken from my GitHub bio; looks fine.

File: src/content/en/2020/http2.md
Comment (rviscomi) [inline src/content/en/2020/http2.md:69]: I see a few more places where `HTTP/1` is used. Should all of those also be converted to `HTTP/1.x` for consistency?

**Review by rviscomi — CHANGES_REQUESTED** [review]

File: src/content/en/2020/http2.md
Comment (MikeBishop) [reply src/content/en/2020/http2.md:69]: Yes, we should take a look and decide whether we're talking about both 1.0 and 1.1, or 1.1 specifically.

**Review by rviscomi — APPROVED** [review]
LGTM. Will let the other coauthors take a look before merging.

**Comment by rviscomi** [comment]
@rmarx @dotjs LGTM?

**Review by rmarx — APPROVED** [review]

**Pull request merged**
